src/test/java/org/springframework/data/r2dbc/function/AbstractDatabaseClientIntegrationTests.java

```diff
@@ -108,6 +109,9 @@ public void executeInsert() {
 				.expectNext(1) //
 				.verifyComplete();
 
+		Flux<LegoSet> rows = databaseClient.select().from("legoset").orderBy(Sort.by(desc("id"))).as(LegoSet.class).fetch()
```

Comment:
Since `rows` doesn't get subscribed to this line doesn't really do much. If it serves a purpose that should be explained in a comment.